### DISCUSSION TITLE
Virtualizationon on iPhone with Apple Silicon A14&A15

### DISCUSSION THREAD

**Discussion by gandizm** [utmapp/UTM#4935] [UTM Help (iOS)] [answered]
Hi,
Will it run hypervisor on  A14&A15/iOS14-15 by using Fugu?
https://worthdoingbadly.com/hv/
As you can see,hv is also available in iPhone12,iOS14 by using Fugu14.(It seems that UTM developers has already known that.)
Fugu15 has already released,and it might be possible to run UTM.HV on iOS15(by unlocking the hypervisor support after jailbreak).
As https://github.com/utmapp/UTM/discussions/3230#discussioncomment-3610391 tried zhuowei's way,I'd like to ask that what has been done in UTM.HV for M1 iPad according to the article I showed on the top , and the concept to run it on iPhone(which seems really cool.)
Thanks

**Comment by osy** [comment, marked as answer]
Someone has to port the HV stuff then sure.

**Reply by gandizm** [reply]
Maybe it is only possible in A14/iOS14 and need fugu14.It seems that running virtualization won’t be supported.
